app/templates/policies.hbs

```diff
@@ -42,6 +42,24 @@
   Code of Conduct.
 </p>
 
+<h3 id='delete-crate'><a href='#delete-crate'>How can I delete a crate I own from the registry?</a></h3>
+
+<p>
+  You can't delete crates from the registry but you could leave it open for
+  ownership transfering.
+</p>
+
+<p>
+  To do this, you must yank all of the versions in the crate
```

Comment:
I'm not sure if yanking is strictly necessary. IMHO there is nothing wrong with keeping the old versions around in case other apps or libraries depend on them.

Reply:
Makes sense to me! I'm cc'ing @carols10cents because we discussed this on chat before.

Reply:
I took the liberty of updating the PR :)